src/components/FormSections/PoliticalParty.jsx

```diff
@@ -12,7 +12,6 @@ function PoliticalParty(props){
 
     //Drupal field data
     const partyField = fields.find(item => item.uuid === "fd516f06-11bb-4c39-9080-735ed98100cc");
-    const partyGeneralInstructions = sanitizeDOM(partyField.instructions)
```

Comment:
Revert this - we want the general instructions to appear if there are no state specific instructions to override